test/cog/chat/slack/templates/embedded/help_bundle_test.exs

```diff
@@ -8,6 +8,13 @@ defmodule Cog.Chat.Slack.Templates.Embedded.HelpBundleTest do
                              "commands" => [%{"name" => "test-command",
                                               "description" => "does just one thing"}],
                              "author" => "vanstee",
+                             "config_file" => %{
```

Comment:
I'm not sure I understand. Are you saying that we should add virtual fields for the config bits? I don't think that would change the test here, would it?